plugins/amazonq/src/main/resources/META-INF/plugin.xml

```diff
@@ -86,6 +86,8 @@
     <extensions defaultExtensionNs="com.intellij">
         <registryKey key="amazon.q.endpoint" description="Endpoint to use for Amazon Q"
                      defaultValue="https://codewhisperer.us-east-1.amazonaws.com/" restartRequired="true"/>
+        <registryKey key="inline.completion.rem.dev.use.rhizome" description="Used for Amazon Q to displaying suggestions on remote"
```

Comment:
Could we modify the description or add a comment to state that this is a JB specific registry key, not one we've set which needs to be turned off for our suggestions to work